Oblig 2/Oppgave 2.ipynb

```diff
@@ -18,13 +18,25 @@
   },
```

Comment:
<p>Nei, du ser nok litt feil på p-verdien: </p><p> </p><p><code>2.629309715803535e-16</code></p><p>e-16 indikerer at dette er et veldig lite tall og derfor i høyeste grad en signifikant forskjell mellom de to modellene.</p><p> </p><p>Forøvrig godt at du bruker sunn fornuft og ser ut fra plottet at det må være noe galt med p-verdien. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Bsingstad/DMBA2024/pull/13/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='20'/>